querydsl-core/src/test/java/com/querydsl/core/types/dsl/ExpressionsTest.java

```diff
@@ -57,6 +57,11 @@ public void  Signature() throws NoSuchMethodException {
                 assertReturnType(Expressions.class.getMethod(type + "Template", Class.class, Template.class, ImmutableList.class));
             }
         }
+
+        // arrays
+        assertReturnType(Expressions.class.getMethod("arrayPath", Class.class, String.class));
+        assertReturnType(Expressions.class.getMethod("arrayPath", Class.class, Path.class, String.class));
+        assertReturnType(Expressions.class.getMethod("arrayPath", Class.class, PathMetadata.class));
```

Comment:
Is there a reason to make sure the return type is the same as the name of the method? It just seems like an odd thing to test.